layers/+tools/google-calendar/README.org

```diff
@@ -0,0 +1,166 @@
+#+TITLE: Google calendar layer
```

Comment:
Depends if there are reliable packages that provides synchronization with other popular calendar services. Non Google calendar users could just exclude `org-gcal` in their dotfile and only use calfw.